docker-compose.yml

```diff
@@ -93,8 +93,9 @@ services:
       args:
         FWATCHDOG_VERSION: "0.7.1"
     volumes:
-      - vendor:/var/www/vendor
-      - logs:/var/www/logs
+      - ./vendor:/var/www/vendor:rw
+      - ./logs/www:/var/www/logs:rw
+      - ./logs/var:/var/log/:rw
       - .:/var/www
```

Comment:
While you're in here, can you go ahead and add the rw to the `var/www` one

Reply:
I wasn't sure which is why I'd left it. ✔️